libraries/AP_GPS/AP_GPS_UAVCAN.cpp

```diff
@@ -74,12 +74,21 @@ void AP_GPS_UAVCAN::subscribe_msgs(AP_UAVCAN* ap_uavcan)
     }
 }
 
+// The Intended behaviour of this method is to ensure that if Two GPSes are connected
+// the boot up is always in order, but in scenarios where one of the GPS is removed, the other
+// GPS whatever be its sequence in the list, is automatically moved up to be the first GPS.
```

Comment:
You can't move it up to the first GPS as it breaks the `POS` parameters, you'd have to move those as well, which can lead to large external state inconsistencies (and makes it harder for a user setting up their vehicle)